website/www/site/data/en/pillars.yaml

```diff
@@ -12,7 +12,13 @@
 
 - title: Unified
   body: Use a single programming model for both batch and streaming use cases.
-- title: Portable
-  body: Execute pipelines on multiple execution environments.
+  icon: icons/unified-icon.svg 
 - title: Extensible
   body: Write and share new SDKs, IO connectors, and transformation libraries.
+  icon: icons/extensive-icon.svg
+- title: Portable
+  body: Execute pipelines on multiple execution environments.
+  icon: icons/portable-icon.svg
+- title: Open Source
+  body: Use a single programming model for both batch and streaming use cases.
```

Comment:
Is this a typo? This is the same body as "Unified"

Reply:
I was following the design. I believe it's just the text placeholder right now, and the copies will be updated later.